src/Summary/SummaryManager.php

```diff
@@ -123,4 +142,38 @@ public function removeLanguage(Adherent $adherent, Language $language): bool
 
         return true;
     }
+
+    public function setUrlProfilePicture(Summary $summary): void
+    {
+        $urlProfilePicture = '#';
+
+        if ($summary->hasPictureUploaded()) {
+            $cache = substr(md5((new \DateTime())->format('U')), 0, 20);
```

Comment:
Risky! What if two people upload their profile pictures at the exact same time? Better to use a function like `uniqid()` or similar to generate a random number.